### DISCUSSION TITLE
Can I use mkcert for local addresses such as 192.168.0.104?

### DISCUSSION THREAD

**Discussion by tuket** [FiloSottile/mkcert#455] [Q&A]
Hi,
Can I use mkcert for local addresses such as 192.168.0.104?

**Comment by j1elo** [comment]
Yes; for example I run this on Linux to get certs for all the listed addresses and domains in the current directory:

```sh
CAROOT="$PWD" mkcert -cert-file cert.pem -key-file key.pem \
    "127.0.0.1" \
    "::1"       \
    "localhost" \
    "*.home.arpa"
```

So it definitely works. You would just do something similar, like running `mkcert 192.168.0.104`.

**Comment by t3dium** [comment]
Yeah i also run this for private ip's, works fine. That's the main usecase of mkcert even
